acceptance/acceptance_test.go

```diff
@@ -209,6 +209,11 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 		t.Skipf("Disabled via GOOS.%s setting in %s", runtime.GOOS, configPath)
 	}
 
+	cloudEnv := os.Getenv("CLOUD_ENV")
+	if config.LocalOnly && cloudEnv != "" {
+		t.Skipf("Disabled via LocalOnly setting in %s (CLOUD_ENV=%s)", configPath, cloudEnv)
+	}
```

Comment:
I suspect that the presence of `config.Server` implies the local-only mode?

Reply:
Interesting thought. I'd rather each option to be independent, even if in practice the correlate.

However, in case of server I don't see that -- you might need some stubs to run it locally but it already works with cloud.